HACKTOBERFEST.md

```diff
@@ -0,0 +1,47 @@
+
+Read more about us [here](https://publiclab.org/about).
+
+## Hacktoberfest Issues
+All contributors, please keep in mind the following points before working on particular issues:
+- All issues that will be considered a part of Hacktoberfest will have the "__Hacktoberfest__" label.
+- If you find an issue you are interested in working on, please comment on the issue so that we and any future contributors are aware. This is necessary so that at the end of the day, two people don’t work on the same issue and submit two different PRs.
+- If there has been no recent activity on an issue for some time, ping the person who had last claimed it, asking if they are still working on it. If they aren’t, you are free to take up the issue for yourself!
+
+We also encourage you to raise new issues! We constantly strive to improve PublicLab, and you can help by pointing out issues.
+- Make sure that you abide by the new issue guideline that is available when you open one.
+- Provide as much information about the bug you can. If it is a UI bug, please provide some screenshots. If there is some issue while setting it up on your local system, provide server logs so that we can help out!
+- If you notice an issue, we encourage you to open it first and get it approved by the maintainers. Opening a PR without doing this may result in it being marked as spam.
+
+If there are any other problems you face - for example, while setting up the project on your local system - the fastest way to resolve it would be by dropping a message on the Gitter chat. Someone from the community will respond ASAP. Raising an issue on GitHub for these reasons is generally discouraged.
+
+If you face any issues with the local installation, try using Gitpod instead!
+
+__If there are any security issues that you uncover, DO NOT post them on GitHub. Send a message on Gitter saying that you have found a security issue, and one of the maintainers will personally message you.__
+
+## Hacktoberfest Pull Requests
+Before making any PR, make sure you read the contributing docs (CONTRIBUTING.md).
+- Fork the PublicLab repo that you want to work on.
+- Make sure that you work on a new branch for every new issue. That way, it is easier to manage various PRs simultaneously.
+
+While making a PR, keep in mind the following:
+- The PR must contain commits you made yourself. We will not accept automated commits from bots.
+- We only accept changes solving a particular issue and the specific code changes mentioned within the issue. We will not merge any change outside of this scope.
+- If you notice an issue, we encourage you to open it first and get it approved by the maintainers. Opening a PR without doing this may result in it being marked as spam.
```

Comment:
Hey @anirudhprabhakaran3 looks great :tada: just wanted to ask this line "if you notice..." is repeated above too. Is this intentional? :sweat_smile:

Reply:
Aah yes, I didn't spot that one. No, it was unintentional. I'll change the wording a bit 😓